docs/website/01_framework/dirichletBCs.md

```diff
@@ -24,23 +24,25 @@ The interface of `#!cpp Ikarus::DirichletValues` is represented by the following
 Ikarus::DirichletValues dirichletValues2(basis); // (1)!
 void fixBoundaryDOFs(f); // (2)!
 void fixDOFs(f); // (3)!
-void fixIthDOF(i); // (4)!
+void setSingleDOF(i, bool flag); // (4)!
```

Comment:
You only added one type name. I'm not sure how we want to have this below there is `size_t` but above `basis` does not have a type name.

Reply:
I think we can have a mixed approached, add the type where the type matters, e.g. for `f` and `basis` its clear what the type is and the type can also be not be easily written, for `flag` i think its importatant to know that you have to pass in a bool. 
On the other hand we have the class reference